sources/source-Send_to_PushBullet.module

```diff
@@ -103,5 +134,48 @@ class Send_to_PushBullet extends superfecta_base {
 		curl_setopt($curl, CURLOPT_HEADER, FALSE);
 		$response = curl_exec($curl);
 		curl_close($curl);
+		$this->DebugPrint("Complete curl array: \n");
+		print_r($queryData);
+				
 	}
+
+	function reformat_cname($content) {
+		$this->DebugPrint("Initial data: \n");
+		$this->DebugPrint("<pre>".$content."</pre>");
+
+		$this->DebugPrint("Converting to UTF-8: ");
+		$content = $this->utf_convert_string($content);
+
+		$this->DebugPrint("UTF-converted data: \n");
+                $this->DebugPrint("<pre>".$content."</pre>");
+
+		$this->DebugPrint("Removing double whitespace, result: \n");
+		$content = $this->remove_double_whitespace($content);
+		$this->DebugPrint("<pre>".$content."</pre>");
+
+		return $content;
+	}
+
+	function utf_convert_string($content) {
+	    if(!mb_check_encoding($content, 'UTF-8')
```

Comment:
Need to check to make sure mb_string is install on the system otherwise fatal errors will occur during dialplan potentially hanging up on calls